WIP/Source/Veazy/src/test/java/vn/edu/fu/veazy/core/service/LessonServiceTest.java

```diff
@@ -0,0 +1,377 @@
+        }
+    }
+
+    @Test
+    public void testUpdateLesson3() throws Exception {
+        try {
+            lessonService.updateLesson(-1, updateForm);
+        } catch (Exception ex) {
+            Assert.assertEquals(new Exception("requester isn't creator"), ex);
+        }
+    }
+
+    @Test
+    public void testUpdateLesson4() throws Exception {
+        try {
+            lessonService.updateLesson(-1, null);
+        } catch (Exception ex) {
+            Assert.assertEquals(new Exception("lesson doesn't exist"), ex);
+        }
+    }
+
+    @Test
+    public void testUpdateLesson5() throws Exception {
+        try {
+            lessonService.updateLesson(null, updateForm);
+        } catch (Exception ex) {
+            Assert.assertEquals(new Exception("requester isn't creator"), ex);
+        }
+    }
+
+    @Test
+    public void testUpdateLesson6() throws Exception {
+        try {
+            lessonService.updateLesson(null, null);
+        } catch (Exception ex) {
+            Assert.assertEquals(new Exception("lesson doesn't exist"), ex);
+        }
+    }
+
+    @Test
+    public void testUpdateLesson7() throws Exception {
+        try {
+            lessonService.updateLesson(learnerId, updateForm);
+        } catch (Exception ex) {
+            Assert.assertEquals(new Exception("requester isn't creator"), ex);
+        }
+    }
+
+    @Test
+    public void testUpdateLesson8() throws Exception {
+        try {
+            lessonService.updateLesson(learnerId, null);
+        } catch (Exception ex) {
+            Assert.assertEquals(new Exception("lesson doesn't exist"), ex);
+        }
+    }
+
+//    @Test
+//    public void testPublishLessonVersion() throws Exception {
+//        try {
+//            lessonService.publishLessonVersion(creatorId, lessonId);
+//        } catch (Exception ex) {
+//            Assert.assertEquals(new Exception("don't have version can be published"), ex);
+//        }
+//    }
+    @Test
+    public void testPublishLessonVersion2() throws Exception {
+        try {
+            lessonService.publishLessonVersion(creatorId, -1);
+        } catch (Exception ex) {
+            Assert.assertEquals(new Exception("lesson doesn't exist"), ex);
+        }
+    }
+
+    @Test
+    public void testPublishLessonVersion3() throws Exception {
+        try {
+            lessonService.publishLessonVersion(creatorId, null);
+        } catch (Exception ex) {
+            Assert.assertEquals(new Exception("lesson doesn't exist"), ex);
+        }
+    }
+
+    @Test
+    public void testPublishLessonVersion4() throws Exception {
+        lessonService.publishLessonVersion(creatorId, lessonId);
+		LessonModel lesson = lessonService.getLesson(lessonId, false);
+		Assert.assertNotEquals(null, lesson.getCurrentVersionId());
+    }
+
+    @Test
+    public void testReportLesson() throws Exception {
+        lessonService.reportLesson(learnerId, lessonId, "content");
```

Comment:
Report hiện tại đang ko có method nào để tôi lấy lại được nên ko so sánh đc

On Fri, Aug 19, 2016 at 6:04 PM, Nguyễn Ngọc Minh <notifications@github.com>
wrote:

> In WIP/Source/Veazy/src/test/java/vn/edu/fu/veazy/core/
> service/LessonServiceTest.java
> <https://github.com/sangnvus/201605JS01/pull/25#discussion_r75464058>:
>
> > +            lessonService.publishLessonVersion(creatorId, null);
> > +        } catch (Exception ex) {
> > +            Assert.assertEquals(new Exception("lesson doesn't exist"), ex);
> > +        }
> > +    }
> > +
> > +    @Test
> > +    public void testPublishLessonVersion4() throws Exception {
> > +        lessonService.publishLessonVersion(creatorId, lessonId);
> > +		LessonModel lesson = lessonService.getLesson(lessonId, false);
> > +		Assert.assertNotEquals(null, lesson.getCurrentVersionId());
> > +    }
> > +
> > +    @Test
> > +    public void testReportLesson() throws Exception {
> > +        lessonService.reportLesson(learnerId, lessonId, "content");
>
> assert
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sangnvus/201605JS01/pull/25/files/442cb029a2ed3ce65a0c4e6062ce71d64bbc74d1#r75464058>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASfcYznu-5nI93DPUGGLMR4PSmXVM2zpks5qhY2lgaJpZM4JnxSY>
> .
>